source/transactions/tests/write-concern.yml

```diff
@@ -305,14 +303,7 @@ tests:
             - name: insertOne
               arguments:
                 document: {_id: 1}
-        result:
-          deletedCount: 0
-          insertedCount: 1
-          insertedIds: {0: 1}
-          matchedCount: 0
-          modifiedCount: 0
-          upsertedCount: 0
-          upsertedIds: {}
+        # avoid assertion since result fields are inaccessible for unacknowledged write concern
```

Comment:
This command is part of a transaction so the driver ignores the w:0 write concern. A result *is* returned.